src/toil_batch_system_tes/tes_batch_system.py

```diff
@@ -71,6 +72,7 @@ def get_default_tes_endpoint(cls) -> str:
 
     def __init__(self, config: Config, maxCores: float, maxMemory: int, maxDisk: int) -> None:
         super().__init__(config, maxCores, maxMemory, maxDisk)
+        set_log_level(config.logLevel, logger)
```

Comment:
I think if we want to use `set_log_level()` on several loggers, we probably want to change it on the Toil side to not do all its logger suppression each time?

Reply:
Probably. It would also be nice if setting the log level explicitly wasn't necessary. In my opinion, logging with plugins should be enabled on default.